src/group_theory/subgroup/basic.lean

```diff
@@ -1678,13 +1678,13 @@ variable {H}
   g ∈ normalizer H ↔ ∀ n, n ∈ H ↔ g * n * g⁻¹ ∈ H := iff.rfl
```

Comment:
While you're here, how about making this:
```suggestion
  g ∈ H.normalizer ↔ ∀ h, h ∈ H ↔ g * h * g⁻¹ ∈ H :=
iff.rfl
```
and moving the new `mem_normalizer_iff''` lemma to be adjacent so it's easy to see the difference at a glance.